modules/virt-marking-vm-templates-favorites.adoc

```diff
@@ -10,7 +10,6 @@ For easier access to virtual machine templates that are used frequently, you can
 
 .Procedure
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtualization* from the side menu.
-. Click the *Templates* tab.
-. Identify the Red Hat template that you want to mark as a favorite.
-. Click the Options menu {kebab} and select *Favorite template*. The template moves up higher in the list of displayed templates.
+. In the {VirtProductName} console, click *Virtualization* -> *Templates* from the side menu.
+
+. Identify the Red Hat template that you want to mark as a favorite and click the Star icon. The template moves up higher in the list of displayed templates.
```

Comment:
Not in the scope of this PR, but I'm confused; I don't see a star icon anywhere in the list of templates/VMs on the test system that I'm currently logged into.

Reply:
It's there, like when you mark a favorite in an app. The star is in the first column, to the left of the template name. It's empty by default. You click it to mark a favorite and the star turns yellow.

Reply:
I changed the text to this:

. Click the star icon next to a template to mark it as a favorite.
+
Favorite templates are displayed at the top of the template list.